dao.md

```diff
@@ -0,0 +1,40 @@
+# 🦞 Senior Blockchain Developer - OceanDAO
+[Ocean Protocol](https://www.oceanprotocol.com/) has a vibrant, expanding ecosystem of engineers, data scientists, neuroscientists, ambassadors, podcasters, that are build traction across 5 pillars of growth, receiving grants and funding to deliver results. To serve them better, we’re hiring!
+
+[OceanDAO](https://www.oceanprotocol.com/dao) is a key component that brings decentralization and scale to participants that are driving traction into the Ocean ecosystem. We’re growing the OceanDAO team. If you love to work with crypto, to build and to engage deeply with an enthusiastic community, we want to hear from you.
+
+
+## Responsibilities
+* You will design & build for OceanDAO core software, from the level of blockchain to the level of dapps. Integrating governance, treasury management, viral mechanisms, and whatever else is needed for OceanDAO to be concretized.
+* The scope of the software is well defined but may evolve over time as DAOs are nascent, e.g. integrating decentralized courts, and decentralizing funding mechanisms. You will co-plan and co-design the product with the OceanDAO team in an agile, open-source development process.
+
+
+## Minimum Qualifications
+* 6+ months of Solidity experience
+* 3+ years backend experience, including NodeJS
+* 4+ years of software experience
+* Experience working in technical/development roles in the crypto projects
+* Process-driven, proactive - able to report on your activity and communicate well with the team and DAO (community).
+
+
+## Nice-to-haves
+* 2+ years working with cloud infrastructure, supporting devops.
+* 1+ years CI/CD experience (e.g. Travis, tox)
+* You’re obsessed with blockchain and DAOs. You play with dapps, vote on proposals, or have contributed code to a DAO.
+* Excellent writing and communication skills. Candidate will engage with the community, receive feedback, identify usage patterns, and communicate macro outcomes.
+* Successfully delivered on promises, comfortable working autonomously and within a team
+* Differentiate good and bad feedback to find real pain points to improve product/program based on learnings and manage and update scripts and reward function accordingly
+* Proven experience in building meaningful technical communities around projects and ability to build and manage relationship with a key stakeholders
```

Comment:
* Manage relationships(plural)
* With key stakeholders, remove the 'a' key stakeholders